<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartScreen</class>
 <widget class="QDialog" name="StartScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>190</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="Auth_form" name="page_login"/>
     <widget class="Reg_form" name="page_reg"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Reg_form</class>
   <extends>QWidget</extends>
   <header>reg_form.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Auth_form</class>
   <extends>QWidget</extends>
   <header>auth_form.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
